app/serializers/merchant_serializer.rb

```diff
@@ -0,0 +1,9 @@
+class MerchantSerializer
+  include FastJsonapi::ObjectSerializer
+  attributes :id, :name
+
+  has_many :items
+  has_many :invoices
+  has_many :invoice_items, through: :invoices
+  has_many :transactions, through: :invoices
```

Comment:
@BrianZanti @dionew1 Are these relationships okay to have in the serializer or is this exposing too much info? Also, our expectations did not have the id in the attributes hash; should it be removed?

Reply:
I wouldn't have these in the serializer unless you need them to be a part of your response.  As for the id, if it will allow you to not have id then I think it's okay to remove it, but I think it may require you to have it but am not 100% sure.